.github/workflows/preview-cleanup.yml

```diff
@@ -33,4 +33,4 @@ jobs:
           number: ${{ github.event.number }}
           id: deploy-preview
           message: > 
-            🪓 PR closed, deleted preview at https://github.com/${{ env.PREVIEW_REPO }}/tree/gh-pages/${{ env.PPREVIEW_PATH }}/
+            🪓 PR closed, deleted preview at https://${{ env.PREVIEW_DOMAIN}}/${{ env.PREVIEW_REPO }}/tree/gh-pages/${{ env.PPREVIEW_PATH }}/
```

Comment:
```suggestion
            🪓 PR closed, deleted preview at https://github.com/gradle/${{ env.PREVIEW_REPO }}/tree/gh-pages/${{ env.PREVIEW_PATH }}/
```

Reply:
Now I got the whole thing 😅 
Sorry.

However, why can't we use the same thing on as we do on `ci`? 🤔 
`https://${{ env.PREVIEW_DOMAIN }}/${{ env.PREVIEW_REPO }}/${{ env.PREVIEW_PATH}}/`
This is actually what I would expect...